.github/scripts/generate_image_maps.py

```diff
@@ -0,0 +1,45 @@
+import itertools
```

Comment:
for follow-ups: Would it make sense to eventually move this (+ the other script) to `neondatabase/dev-actions` ?

Reply:
Yep, that's the plan `scripts/generate_image_maps.py` is a bit different for each repo, but `scripts/push_with_image_map.py` is exactly the same. We aim to move all these common things to `neondatabase/dev-actions`, no ETA so far